docker/constraints.in

```diff
@@ -11,7 +11,7 @@ click>=8.0
 cloudpickle>=0.4
 compliance-checker>=4.0.0
 dask>=2021.5.1
```

Comment:
dask and distributed versions should be the same ideally

Reply:
Thanks. I suspect that will be easier to figure out than the click version issue the tests have thrown up.

Reply:
https://github.com/pallets/click/issues/1736